src/pulselistener/tests/test_hook.py

```diff
@@ -1,55 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
I think it would be useful to keep the create_task and create_task_env tests (converting them to use HookCodeCoverage).

Reply:
Basically, testing HookCodeCoverage::run.